py/desitarget/cuts.py

```diff
@@ -255,91 +256,72 @@ def isLRGpass(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=None,
     return lrg, lrg1pass, lrg2pass
 
 
-def isELG(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=None,
-          gallmask=None, rallmask=None, zallmask=None, brightstarinblob=None,
-          south=True, primary=None):
+def isELG(gflux=None, rflux=None, zflux=None, gsnr=None, rsnr=None, zsnr=None,
+          maskbits=None, south=True, primary=None):
     """Definition of ELG target classes. Returns a boolean array.
-
-    Args:
-        gflux, rflux, zflux, w1flux, w2flux: array_like
-            The flux in nano-maggies of g, r, z, w1, and w2 bands.
-        gallmask, rallmask, zallmask: array_like
-            Bitwise mask set if the central pixel from all images
-            satisfy each condition in g, r, z.
-        brightstarinblob: boolean array_like or None
-            ``True`` if the object shares a blob with a "bright" (Tycho-2) star.
-        south: boolean, defaults to ``True``
-            Use cuts appropriate to the Northern imaging surveys (BASS/MzLS) if ``south=False``,
-            otherwise use cuts appropriate to the Southern imaging survey (DECaLS).
-        primary: array_like or None
-            Set to ``True`` for objects to initially consider as possible targets.
-            Defaults to everything being ``True``.
-    Returns:
-        mask : array_like. True if and only if the object is an ELG
-            target.
+    (see :func:`~desitarget.cuts.set_target_bits` for parameters).
 
     Notes:
-    - Current version (08/01/18) is version 144 on `the wiki`_.
+    - Current version (06/25/19) is version 180 on `the wiki`_.
     """
     if primary is None:
         primary = np.ones_like(rflux, dtype='?')
     elg = primary.copy()
 
-    elg &= notinELG_mask(gallmask=gallmask, rallmask=rallmask, zallmask=zallmask,
-                         brightstarinblob=brightstarinblob, primary=primary)
+    elg &= notinELG_mask(maskbits=maskbits, gsnr=gsnr, rsnr=rsnr, zsnr=zsnr,
+                         primary=primary)
 
-    elg &= isELG_colors(gflux=gflux, rflux=rflux, zflux=zflux, w1flux=w1flux, w2flux=w2flux,
-                        south=south, primary=primary)
+    elg &= isELG_colors(gflux=gflux, rflux=rflux, zflux=zflux, south=south,
+                        primary=primary)
 
```

Comment:
Yeah, these changes broke the mocks.  Even though `w1flux` and `w2flux` aren't used for ELG target selection, the way the targeting cuts are applied to the mocks is to pass all the *grz* photometry independent of the class of object.

Reply:
Oops, my bad. I should have remembered *that*. I'll restore this branch and fix that in the next 20 minutes.

Reply:
Actually, I'll just fix this in a separate PR for better transparency. I'll ask for your review in the next PR.

Reply:
I have the changes in a branch already and I'm testing other permutations of the code. I can open a simple PR (or even push the very minor change to master...).  Let me know.

Reply:
Will fix in #514. Thanks.